### PR TITLE
Remove ng modules

### PR DESCRIPTION
**Pull request by timostamm**
Based on https://github.com/connectrpc/examples-es/pull/2246.

### CODE REVIEW

File: angular/src/app/app.component.ts [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
 @Component({
-  imports: [CommonModule, FormsModule],
   selector: "app-root",
-  templateUrl: "./app.component.html",
+  imports: [CommonModule, FormsModule],
   styleUrls: ["./app.component.css"],
+  templateUrl: "./app.component.html",
 })
 export class AppComponent {
+  client = inject(ELIZA);
```
Comment (timostamm) [inline angular/src/app/app.component.ts:18]: Instead of `@Inject`, we use the function `inject`.

File: angular/src/connect/connect.module.ts [hunk 2]
```diff
@@ -1,29 +1,63 @@
...
+export const TRANSPORT = new InjectionToken<Transport>("connect.transport");
+
+export const INTERCEPTORS = new InjectionToken<Interceptor[]>(
+  "connect.interceptors",
+  {
+    factory: () => [],
+  },
+);
+
+export function createClientToken<T extends DescService>(service: T): InjectionToken<ObservableClient<T>> {
```
Comment (timostamm) [inline angular/src/connect/connect.module.ts:16]: This just makes it a bit easier to set up clients for many services.

**Review by timostamm — COMMENTED** [review]
Haven't updated the tests.

File: angular/src/main.ts [hunk 3]
```diff
@@ -1,12 +1,18 @@
-import { enableProdMode } from '@angular/core';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+/*
+ *  Protractor support is deprecated in Angular.
+ *  Protractor is used in this example for compatibility with Angular documentation tools.
```
Comment (smaye81) [inline angular/src/main.ts:3]: I'm not sure we should add a deprecated feature if we're trying to adhere to best practices in the example. If we're worried about users being confused looking at the example, can we just add a simple comment and not include the import and provider?

File: angular/src/connect/tokens.ts [hunk 4]
```diff
@@ -1,17 +1,4 @@
...
-  "connect.interceptors",
-  {
-    factory: () => [],
-  },
-);
-
-export const ELIZA = new InjectionToken<ObservableClient<typeof ElizaService>>(
-  ElizaService.name,
-);
+export const ELIZA = createClientToken(ElizaService);
```
Comment (smaye81) [inline angular/src/connect/tokens.ts:4]: Can we move this to `connect.module.ts`? Feels odd to have it it in its own `tokens` file now.

File: angular/src/main.ts (hunk 3 above)
Comment (timostamm) [reply angular/src/main.ts]: This was blindly copied from https://angular.dev/tutorials/first-app 
Yes, we should remove it.

This also explains why the tutorial had an unused app.config.ts file. We can put the provider setup there. Done in https://github.com/connectrpc/examples-es/pull/2247/commits/83a1f0e9f5d9bc0b94534e65d32b90b794ca1f6f

File: angular/src/connect/connect.module.ts [hunk 5]
```diff
@@ -1,29 +1,56 @@
...
-              interceptors: interceptors,
-            }),
-          deps: [INTERCEPTORS],
-        },
-      ],
-    };
-  }
+const TRANSPORT = new InjectionToken<Transport>("connect.transport");
+
+const INTERCEPTORS = new InjectionToken<Interceptor[]>("connect.interceptors", {
```
Comment (timostamm) [inline angular/src/connect/connect.module.ts:12]: Users can't add an interceptor if you don't export this const.

**Review by smaye81 — APPROVED** [review]

**Pull request merged**